autoscale-dockerswarm-shared/src/test/java/com/hpe/caf/autoscale/shared/endpoint/docker/DeveloperEndpointIT.java

```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2015-2017 EntIT Software LLC, a Micro Focus company.
```

Comment:
Check this file and the others where new headers appear to have been added.  I wouldn't expect new headers in open-source files since we are already enforcing that all files have headers.

Reply:
Looks like something went wrong with the mvn license format command where instead of replacing headers it is adding a new one. Will review this now.